src/app/tickets/_components/TicketOpportunityRow.module.css

```
@@ -11,6 +13,34 @@
+   from the tab order) - same technique as
+   src/ui/TriStateCheckbox.module.css's own .input. The row link has no
+   other visible text of its own (the visible date/title/badges below are
+   plain, non-link text), so this is its only accessible name. */
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)